frontend/src/app/scripts/_components/ScriptAccordion.tsx

```diff
@@ -104,13 +104,13 @@ export default function ScriptAccordion({
                   >
                     <div className="flex items-center">
                       <Image
-                        src={script.logo || "/logo.png"}
+                        src={script.logo || "/ProxmoxVE/logo.png"}
```

Comment:
Don't we have a `basePath` variable we can use here?

Reply:
totally forgot, fixed.